tests/conftest.py

```diff
@@ -72,11 +73,10 @@ def updated_global_config(request: FixtureRequest, admin_client: DynamicClient)
     else:
         py_config["distribution"] = "downstream"
```

Comment:
if we want to make sure the distribution names in the DSC don't change without us noticing, I would update this to 
```
elif get_operator_distribution(client=admin_client) == "OpenShift AI":
    py_config["distribution"] = "downstream"
else:
    raise UnknownDistribution (or pytest.fail(f"Unknown distribution value: {get_operator_distribution(client=admin_client)}") or LOGGER.error(f"Unknown distribution value: {get_operator_distribution(client=admin_client)}", defaulting to downstream) etc.)
```

Reply:
Not necessary. By default we have distribution as "downstream" we only set it to "upstream" in updated_global_config() if the dsc tell's us that it is open data hub. No other values are possible.

Reply:
What I mean is that `get_operator_distribution` fetches the value from the DSC, and if there's a new value we don't know of it will set this to "downstream" when we maybe didn't want it to. It would be better to check explicitly that it's returning openshift ai before setting it to downstream.

Reply:
```   
if get_operator_distribution(client=admin_client) == "Open Data Hub":
        py_config["distribution"] = "upstream"

    else:
        py_config["distribution"] = "downstream"
``` this is coming from updated_global_config. If you believe it is a possibility to have a different value, then we should set it there. I will add it.